specs/sb/api.md

```diff
@@ -342,6 +400,45 @@ A web-friendly display name is camel-cased with spaces and punctuation supported
         }, {
           "content": "40 concurrent connections"
         }]
+      },
+      "schemas": {
+        "service_instance": {
+          "parameters": {
+            "required": [
+              "fake_required_property"
+            ],
+            "properties": {
+              "fake_required_property": {
+                "type": "string"
+              }
+            }
+          },
+          "outputs": {
+            "properties": {
```

Comment:
Your observation for service instances outputs is correct. However it is within scope to schematize outputs for service bindings, and I didn't want to define separate objects for each. I can either separate the objects into nearly identical objects, or specify in the docs that outputs schema is only valid for bindings. How do these approaches seem to you?